src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/ConfigureEndpointRabbitMQTransport.cs

```diff
@@ -43,14 +41,14 @@ void PurgeQueues()
 
         var queues = transport.QueuesToCleanup.Distinct().ToArray();
```

Comment:
Consider using [`Enumerable.ToHashSet` Method](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.tohashset)

```suggestion
        var queues = transport.QueuesToCleanup.ToHashSet();
```